tests/conftest.py

```diff
@@ -74,7 +79,8 @@ def mock_account(api_url, account_id):
             "name": "Ben Bitdiddle",
             "object": "account",
             "provider": "gmail",
-            "organization_unit": "label"
+            "organization_unit": "label",
+            "billing_state": "trial",
```

Comment:
What happens if you call `.upgrade()` on an account in `paid` status? What happens if you call `.downgrade()` on an account in `cancelled` or `deleted` status?